azavea/cac-tripplanner
# Feature/kak/events sidebar#1126

## Overview

Implement map page for events.

Do not route to event destinations, but present them as cards in the sidebar, similar to the 'tour' sidebar, but with a slightly different header. On the map, show markers for the event destinations.


### Demo

![image](https://user-images.githubusercontent.com/960264/66340114-874a7e00-e912-11e9-90d9-9643978805be.png)


### Notes

This uses the same sidebar control and elements as the `tour` sidebar, which is why it also has a fixed height as a temporary fix for scrolling.


## Testing Instructions

 * Clicking an event card from the home page should go to event map view
 * Clicking an event card from the map sidebar 'explore' cards should go to event map view
 * Selecting an event from the destination autocomplete options should go to the event map view
 * Event destination map markers should display and map should zoom to fit them all
 * Clicking a destination card from the event sidebar should go to directions to that place
 * Clicking the browser back button from destination directions should return to event map view
 * 'More info' destination card button should link to place detail page
 * On sidebar card hover, 'directions' button should highlight and map marker circle should appear


Closes #1126
